CMakeLists.txt

```diff
@@ -381,10 +373,25 @@ if (NOT gismo_optionals STREQUAL "")
   message(STATUS "Optional compile list: ${gismo_optionals}")
 endif()
 
-if(GISMO_UNSUPPORTED)
+## #################################################################
+## Submodules
+## #################################################################
+
+if(GISMO_UNSUPPORTED OR GISMO_MOTOR)
   gismo_fetch_module(unsupported)
+endif()
+
+if(GISMO_MOTOR)
   gismo_fetch_module(motor)
 endif()
 
+if(GISMO_ELASTICITY)
+  gismo_fetch_module(gsElasticity)
+endif()
+
+if(GISMO_EXASTENCILS)
+  gismo_fetch_module(gsExaStencils)
+endif()
+
```

Comment:
@filiatra this code you mentioned is still there

Reply:
its about the order. that might matter.